src/aosm/azext_aosm/configuration_models/sns_parameters_config.py

```diff
@@ -0,0 +1,18 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from __future__ import annotations
+
+from abc import ABC
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class SNSCommonParametersConfig(ABC):

```

Comment:
OnboardingSNSInputConfig has location, operator_resource_group_name, site_name, what is the purpose of SNSCommonParametersConfig, and do you see this being common for any other workflow?

Reply:
Yes Priya, this is the format they have used for all the other workflows, different Configs for all parameters output and to read it and for other use cases